robot/description/launch/caffeine_gazebo.launch

```diff
@@ -1,19 +1,33 @@
 <launch>
    <arg name="use_gui" default="true"/>
    <arg name="debug_mode" default="false"/>
-   <arg name="world" default="$(find husky_gazebo)/worlds/clearpath_playpen.world"/>
-
-   <!-- Initial Pose (NOTE: angles in radians) -->
-   <arg name="x" value="0"/>
-   <arg name="y" value="0"/>
-   <arg name="z" value="0"/>
-   <arg name="roll" value="0"/>
-   <arg name="pitch" value="0"/>
-   <arg name="yaw" value="0"/>
-   
+   <arg name="world" default="walls"/>
+   <arg name="spawn" default="start"/>
+
+   <!-- IGVC worlds -->
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_walls.world"  if="$(eval arg('world') == 'walls')"/>
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_ramp.world"   if="$(eval arg('world') == 'ramp')"/>
+   <arg name="world_file" value="$(find worlds)/gazebo_worlds/igvc_plain.world"  if="$(eval arg('world') == 'plain')"/>
+
+   <!-- Default spawn at course start -->
+   <arg name="x"     value="0"      if="$(eval arg('spawn') == 'start')"/>
+   <arg name="y"     value="-21.5"  if="$(eval arg('spawn') == 'start')"/>
+   <arg name="z"     value="0.3"    if="$(eval arg('spawn') == 'start')"/>
+   <arg name="roll"  value="0"      if="$(eval arg('spawn') == 'start')"/>
+   <arg name="pitch" value="0"      if="$(eval arg('spawn') == 'start')"/>
+   <arg name="yaw"   value="3.14"   if="$(eval arg('spawn') == 'start')"/>
+
+   <!-- Spawn at ramp -->
+   <arg name="x"     value="-19.4"  if="$(eval arg('spawn') == 'ramp')"/>
+   <arg name="y"     value="-12.7"  if="$(eval arg('spawn') == 'ramp')"/>
+   <arg name="z"     value="0.24"   if="$(eval arg('spawn') == 'ramp')"/>
+   <arg name="roll"  value="0"      if="$(eval arg('spawn') == 'ramp')"/>
+   <arg name="pitch" value="0"      if="$(eval arg('spawn') == 'ramp')"/>
```

Comment:
We can use a `<group>` tag to minimize the number of if-statments

Reply:
actually tried group tag first, but ros doesn't let you define arg values more than once within a launch file (unless you have some if/unless statement)